### PR TITLE
ddl: fix data race about ReorgWaitTimeout in tests

### PR DESCRIPTION
**Pull request by zimulala**
### What problem does this PR solve? 
```
WARNING: DATA RACE
Write at 0x000003a867b0 by goroutine 144:
  github.com/pingcap/tidb/ddl_test.(*testDBSuite).TestCancelDropColumn()
      /home/jenkins/workspace/tidb_ghpr_unit_test/go/src/github.com/pingcap/tidb/ddl/db_test.go:1103 +0x3ad
  github.com/pingcap/tidb/session.(*session).Execute()
      /home/jenkins/workspace/tidb_ghpr_unit_test/go/src/github.com/pingcap/tidb/session/session.go:910 +0xd4
  github.com/pingcap/tidb/util/testkit.(*TestKit).Exec()
      /home/jenkins/workspace/tidb_ghpr_unit_test/go/src/github.com/pingcap/tidb/util/testkit/testkit.go:137 +0xff
  github.com/pingcap/tidb/util/testkit.(*TestKit).MustExec()
      /home/jenkins/workspace/tidb_ghpr_unit_test/go/src/github.com/pingcap/tidb/util/testkit/testkit.go:171 +0x94
  github.com/pingcap/tidb/ddl_test.(*testDBSuite).mustExec()
      /home/jenkins/workspace/tidb_ghpr_unit_test/go/src/github.com/pingcap/tidb/ddl/db_test.go:1585 +0x86
  github.com/pingcap/tidb/ddl_test.(*testDBSuite).TestCancelDropColumn()
      /home/jenkins/workspace/tidb_ghpr_unit_test/go/src/github.com/pingcap/tidb/ddl/db_test.go:1088 +0x296
  github.com/pingcap/tidb/util/testkit.(*TestKit).MustExec()
      /home/jenkins/workspace/tidb_ghpr_unit_test/go/src/github.com/pingcap/tidb/util/testkit/testkit.go:171 +0x94
  github.com/pingcap/tidb/ddl_test.(*testDBSuite).mustExec()
      /home/jenkins/workspace/tidb_ghpr_unit_test/go/src/github.com/pingcap/tidb/ddl/db_test.go:1585 +0x86
  github.com/pingcap/tidb/ddl_test.(*testDBSuite).TestCancelDropColumn()
      /home/jenkins/workspace/tidb_ghpr_unit_test/go/src/github.com/pingcap/tidb/ddl/db_test.go:1087 +0x252
  github.com/pingcap/tidb/util/testkit.(*TestKit).MustExec()
      /home/jenkins/workspace/tidb_ghpr_unit_test/go/src/github.com/pingcap/tidb/util/testkit/testkit.go:171 +0x94
  github.com/pingcap/tidb/ddl_test.(*testDBSuite).TestCancelDropColumn()
      /home/jenkins/workspace/tidb_ghpr_unit_test/go/src/github.com/pingcap/tidb/ddl/db_test.go:1086 +0x20e
  runtime.call32()
      /usr/local/go/src/runtime/asm_amd64.s:522 +0x3a
  reflect.Value.Call()
      /usr/local/go/src/reflect/value.go:308 +0xc0
  github.com/pingcap/check.(*suiteRunner).forkTest.func1()
      /home/jenkins/workspace/tidb_ghpr_unit_test/go/pkg/mod/github.com/pingcap/check@v0.0.0-20190102082844-67f458068fc8/check.go:836 +0xa04
  github.com/pingcap/check.(*suiteRunner).forkCall.func1()
      /home/jenkins/workspace/tidb_ghpr_unit_test/go/pkg/mod/github.com/pingcap/check@v0.0.0-20190102082844-67f458068fc8/check.go:730 +0x89

Previous read at 0x000003a867b0 by goroutine 104:
  github.com/pingcap/tidb/ddl.(*worker).runReorgJob()
      /home/jenkins/workspace/tidb_ghpr_unit_test/go/src/github.com/pingcap/tidb/ddl/reorg.go:129 +0x706
  github.com/pingcap/tidb/ddl.(*worker).onCreateIndex()
      /home/jenkins/workspace/tidb_ghpr_unit_test/go/src/github.com/pingcap/tidb/ddl/index.go:313 +0x9bb
  github.com/pingcap/tidb/ddl.(*worker).runDDLJob()
      /home/jenkins/workspace/tidb_ghpr_unit_test/go/src/github.com/pingcap/tidb/ddl/ddl_worker.go:503 +0x84b
  github.com/sirupsen/logrus.Infof()
      /home/jenkins/workspace/tidb_ghpr_unit_test/go/pkg/mod/github.com/sirupsen/logrus@v1.2.0/exported.go:148 +0x7c
```
### What is changed and how it works?
The `ReorgWaitTimeout` is set in ddl_test.go, we needn't set it again. So I remove it.

### CODE REVIEW

**Comment by zimulala** [comment]
PTAL @eurekaka @crazycs520 @winkyao

**Comment by zimulala** [comment]
PTAL @xiekeyi98

File: table/column.go [hunk 1]
```diff
@@ -372,7 +372,8 @@ func getColDefaultValue(ctx sessionctx.Context, col *model.ColumnInfo, defaultVa
 			t := value.GetMysqlTime()
 			// For col.Version = 0, the timezone information of default value is already lost, so use the system timezone as the default value timezone.
 			defaultTimeZone := timeutil.SystemLocation()
-			if col.Version == model.ColumnInfoVersion1 {
+
+			if col.Version >= model.ColumnInfoVersion1 {
```
Comment (eurekaka) [inline table/column.go]: Is this change related to the data race?

File: ddl/db_test.go [hunk 2]
```diff
@@ -596,9 +596,6 @@ func (s *testDBSuite) TestCancelDropTableAndSchema(c *C) {
 		{true, model.ActionDropSchema, model.JobStateRunning, model.StateDeleteOnly, false},
 	}
 	var checkErr error
-	oldReorgWaitTimeout := ddl.ReorgWaitTimeout
```
Comment (xiekeyi98) [inline ddl/db_test.go]: Why delete these lines rather than add mutex lock?

File: table/column.go (hunk 1 above)
Comment (zimulala) [reply table/column.go]: It's my fault. 
Done.

File: ddl/db_test.go (hunk 2 above)
Comment (zimulala) [reply ddl/db_test.go]: > The `ReorgWaitTimeout` is set in ddl_test.go, we needn't set it again. So I remove it.

Comment (xiekeyi98) [reply ddl/db_test.go]: Got it.
